File: 0/A - Bear and Big Brother.cpp
#include <iostream>
using namespace std;
 
 
int main ()
{
 int Limak, Bob, counter;
 cin>>Limak>>Bob;
 
while(Limak <= Bob){
	Limak *= 3;
	Bob *= 2;
	counter ++;
	
}
 
 cout<<counter;
 
  return 0;
}

File: 0/A - Bear and Big Brother.py
 
 
A = input().split(" ")
 
Limak = int(A[0])
Bob= int(A[1])
 
counter = 0
while Limak <= Bob :
    Limak = Limak * 3
    Bob = Bob * 2
    counter +=1
print(counter)
 